### PR TITLE
Fix relative paths in html plugin

### LINKED ISSUE #16: bug: non root html emit does not work
A config that emits a HTML file like so:

```ts
    html({
      template: createHtmlTemplate,
      title: "React App",
      meta: [
        { charset: "utf-8" },
        {
          name: "viewport",
          content: "width=device-width, initial-scale=1",
        },
        {
          name: "description",
          content: "Web site created using create-react-app",
        },
      ],
      fileName: "x/test.html",
    }),
```

leads to three issues:

1. The emitted `<script src="mod.js" type="module"></script>` should look like `<script src="../mod.js" type="module"></script>`
2. The emitted `<link href="mod.css" rel="stylesheet">` should look like `<link href="../mod.css" rel="stylesheet">`
3. `mkdir -r dist/x` has to be run before denopack, because denopack does not create the directory itself.

**Meta**

```shell
> denopack --version
denopack/0.8.0 linux-x86_64 deno-1.3.3
```

### PR DESCRIPTION
**Pull request by Soremwar**
Also sneaky change, removed .vscode folder from the project

TBH, it's better if everyone has it's own set of settings

Closes #16

### CODE REVIEW

**Comment by Soremwar** [comment]
I have no idea why CI is failing. Apparently `deno fmt --check` is complaining on every line

@lucacasonato some help?

File: util/htmlTemplate.ts [hunk 1]
```diff
@@ -58,14 +75,15 @@ export const htmlTemplate = async ({
...
-  <head>
-    ${metas}
-    <title>${title}</title>
-    ${links}
-  </head>
-${body}
-</html>`;
+  return (
+    `<!doctype html>
+      <html${makeHtmlAttributes(attributes.html)}>
```
Comment (jeroenptrs) [inline util/htmlTemplate.ts:80]: @Soremwar was this suggested by deno fmt? because this will cause unnecessary indenting in the html template

**Comment by jeroenptrs** [comment]
> I have no idea why CI is failing. Apparently `deno fmt --check` is complaining on every line
> 
> @lucacasonato some help?

fyi this was fixed with adding `.gitattributes` in this PR

File: util/htmlTemplate.ts (hunk 1 above)
Comment (Soremwar) [reply util/htmlTemplate.ts:80]: No, it was me

Forgot "`" also took indenting

**Review by jeroenptrs — APPROVED** [review]
Gonna land tests for this in a separate PR before making a release

**Pull request merged**